src/rprblender/engine/export_engine.py

```diff
@@ -36,6 +36,11 @@
 
 class ExportEngine(Engine):
     TYPE = 'EXPORT'
+    ENGINE_LIBS = {
```

Comment:
I don't like the idea to store this constant here. I think better just to add code of getting engine_lib_name inside save_json() function.
So, changes should be applied only in save_json(), nothing else has to be touched.